flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/LegacyMySqlSourceExampleTest.java

```diff
@@ -49,5 +49,6 @@ public void testConsumingAllEvents() throws Exception {
         env.addSource(sourceFunction).print().setParallelism(1);
 
         env.execute("Print MySQL Snapshot + Binlog");
+        inventoryDatabase.dropDatabase();
```

Comment:
Add `DROP DATABASE IF EXISTS xxx;` to the `CREATE_DATABASE_DDL` in `UniqueDatabase`. 
We will not need to add this method at so many places.